go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/golang-jwt/jwt/v4 v4.5.0 // indirect
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/protobuf v1.5.4 // indirect
-	github.com/google/go-github/v62 v62.0.0 // indirect
+	github.com/google/go-github/v62 v62.0.1-0.20240626135736-42ae4fe29180 // indirect
```

Comment:
In this PR, `github.com/google/go-github/v62` is directly imported.
Therefore, you can move this to:
https://github.com/reviewdog/reviewdog/blob/8a2b40e3df97fb91fb87b340472a11575120bb71/go.mod#L5-L32

Reply:
Thanks. Fixed.